Dockerfile

```diff
@@ -26,4 +26,4 @@ WORKDIR /data/app
 ADD . .
 
 EXPOSE 8080
-CMD ["sh", "scripts/container_start.sh"]
+CMD ["sh", "scripts/container_start.sh run "]
```

Comment:
@iamareebjamal How do we pass the command from celery container here?

Reply:
Change this to ENTRYPOINT